client/src/components/NearestHospital/CheckSymptomsBox.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import styled from "styled-components";
+import SubmitButton from "../Button/SubmitButton";
+
+const checkSymptoms = props => {
+
+  const CheckSymptomsBox = styled.div`
```

Comment:
yeah better for performance indeed, everything that can be defined once and doesn't change I usually try to declare it outside of the component, otherwise it will be recreated for every component render